app/templates/direct-award/view-project.html

```diff
@@ -104,7 +104,9 @@
                 data-analytics-action=\"External Link\"
                 >how to award a contract when you buy services</a>."])|safe},
               {"type": "text", "text": ''.join(["The buyer and supplier must both sign a copy of the contract before the service can be used."])|safe},
-              {"type": "box", "style": "inactive", "label": "Can't start yet"} if not project.downloadedAt else {}
+              {"type": "box", "style": "complete", "label": project.outcome} if project.outcome else {},
```

Comment:
So:

1) for non-award outcomes, we need to translate `project.outcome` into a human-readable string, as it's just a machine-readable identifier right now;

2) for award outcomes, you need to check `project.outcome.resultOfDirectAward.archivedServiceId` and look up the details of the service using the `/archived-services/<int:archived_service_id>` endpoint of the API. In this case we'll have to hard-code a reference to the service name question, and document that we've done that in the frameworks repo.